src/pharmpy/tools/mfl/parse.py

```diff
@@ -93,53 +103,120 @@ def validate_mfl_list(mfl_statement_list):
 class ModelFeatures:
     def __init__(
         self,
-        absorption=Absorption((Name('INST'),)),
-        elimination=Elimination((Name('FO'),)),
-        transits=(Transits((0,), (Name('DEPOT'),)),),  # NOTE : This is a tuple
-        peripherals=Peripherals((0,)),
-        lagtime=LagTime((Name('OFF'),)),
+        absorption=None,
+        elimination=None,
+        transits=tuple(),  # NOTE : This is a tuple
+        peripherals=tuple(),
+        lagtime=None,
         covariate=tuple(),  # Note : Should always be tuple (empty meaning no covariates)
+        direct_effect=None,
+        effect_comp=None,
+        indirect_effect=tuple(),
+        metabolite=None,
+        _let={},
```

Comment:
This has to do with the fact that LET() statements were not previously stored in an MFL object. Mainly because a search space could evaluate the reference values given in LET statements immediately and then drop them.

However, this removed the possibility of having a search space of only LET() statements, which could be used if one wanted to overwrite the default search space's reference values (which currently are @IIV). Perhaps this adaptability should be removed, and a search space that only contains LET() statements should be interpreted to be empty.

@rikardn perhaps you have some input? Should the MFL object keep track of LET() and store them on the object or should they be evaluated and dropped upon parsing?